tests/unit/dataactbroker/test_file_handler.py

```diff
@@ -267,8 +268,12 @@ def test_submission_bad_dates(start_date, end_date, quarter_flag, submission):
 
 
 def test_submission_to_dict_for_status(database):
+    for ps in PUBLISH_STATUS:
```

Comment:
There's an easier way to add this by default instead of doing it manually, using fixtures. Take a look at https://github.com/fedspendingtransparency/data-act-broker-backend/blob/development/tests/unit/conftest.py#L49-L51